iphone/Maps/Core/iCloud/SynchronizationFileWriter.swift

```diff
@@ -251,6 +249,23 @@ final class SynchronizationFileWriter {
   }
 }
 
+// MARK: - FileManager + FileReplacing
+private extension FileManager {
+  func replaceFileSafe(at sourceUrl: URL, with targetUrl: URL) throws {
```

Comment:
Should it be swapped?
```suggestion
  func replaceFileSafe(at targetUrl: URL, with sourceUrl: URL) throws {
```

Reply:
Thanks! You're right!